data/companies.yml

```diff
@@ -95,7 +95,7 @@ providers:
     logo: "/logos/microsoft-azure.svg"
     url: "https://learn.microsoft.com/en-us/azure/aks/istio-about"
   - name: "IMESH Platform"
-    logo: "/logos/imesh.svg"
+    logo: "/logos/imesh-logo.svg"
```

Comment:
name this imesh.svg and remove all unrelated changes from the PR

Reply:
Name needs to match the image, which it currently does. Making the change would also require changing he image name.